pkg/cmd/cli/cmd/login/loginoptions.go

```diff
@@ -66,6 +67,7 @@ type LoginOptions struct {
 	PathOptions *kclientcmd.PathOptions
 
 	CommandName string
+	Timeout     string
```

Comment:
and call it RequestTimeout to make it clear it is per-request, not for the whole operation